test_add_user.rb

```diff
@@ -0,0 +1,24 @@
+require 'test-unit'
+require 'selenium-webdriver'
+require_relative 'add_user'
+
+class TestAddUser < Test::Unit::TestCase
+  include AddUser
+
+  def setup
+    @driver = Selenium::WebDriver.for :firefox
+    @wait = Selenium::WebDriver::Wait.new(:timeout => 10)
+  end
+
+  def test_add_user
+    add_user
```

Comment:
when you move your helper methods into one module I'll be able to reuse a lot of action (in case you divide your complex methods into logical parts)

for example method `add_user` doesn't only add user right?
there is a registration of user, creating a new project, registration of another one user, adding him to the project, etc.
we want to have these logical parts to reuse them as much as we can